clients/privacy-center/cypress/e2e/consent-banner.cy.ts

```diff
@@ -946,12 +946,11 @@ describe("Consent banner", () => {
       });
 
       it("closes banner and opens modal when modal link is clicked", () => {
-        cy.get("div#fides-banner").should("exist");
+        cy.get("div#fides-banner").should("be.visible");
```

Comment:
this is a little subtle—the banner always exists, we just move it out of the screen so that we can animate it. the odd thing is that without this change, the env would end up as

<img width="514" alt="image" src="https://github.com/ethyca/fides/assets/24641006/78e6583a-9260-4451-a562-f47780e8baec">

showing _both_ the modal and overlay. I think what is happening is

1. the page loads, the banner does not appear yet because it is still animating in
2. cypress clicks on the modal link, so the modal opens (also sets that the banner should not be open, but I think at this point the banner isn't technically open yet because of [`delayBanner`](https://github.com/ethyca/fides/blob/aking/3485/fix-cy-tests/clients/fides-js/src/components/Overlay.tsx#L40-L45))
3. now the banner has finished animating so both the modal and banner are open!

by waiting for the banner to be visible, that delays cypress enough so that the banner is set to open, then the logic all works accordingly.

this is probably a bug, so maybe it should be skipped instead of the fix I put in 🤔 let me know what you think @eastandwestwind . I think if a user were very speedy, they could trigger this by clicking on the modal link before the banner has appeared

Reply:
I think this cypress adjustment is fine for now, as the odds that a user immediately clicks on consent (usually in the footer of a site) before the banner slides in is slim. 

I'll create a follow-up for this bug fix.

Reply:
https://github.com/ethyca/fides/issues/3488